hw/ip/otbn/doc/_index.md

```diff
@@ -649,13 +668,70 @@ Before being stored in SRAM, the instruction word with the attached Integrity Pr
 The scrambling is reversed on a read.
 
 The ephemeral memory scrambling key and the nonce are provided by the [OTP block]({{<relref "/hw/ip/otp_ctrl/doc" >}}).
-They are set once when OTBN block is reset, and changed whenever a secure wipe of the instruction memory is performed.
-(TODO: Link to it once we have specified secure wipe.)
+They are set once when OTBN block is reset, and changed whenever a [secure wipe]({{<relref "#design-details-secure-wipe-imem">}}) of the instruction memory is performed.
 
 The Integrity Protection Code is checked on every memory read, even though the code remains attached to the data.
 A further check must be performed when the data is consumed.
 Detected integrity violations in the data memory raise a fatal `imem_error`.
 
+### Secure Wipe {#design-details-secure-wipe}
+
+The entire OTBN internal state including the instruction and data memories, can be securely deleted on demand from a host software.  
+In addition, full or partial secure wipe is triggered automatically by the OTBN in certain situations. 
+
+#### Triggering Secure Wipe
+
+In the following situations OTBN itself initiates a full secure wipe:
+* The lifecycle controller asks for it through its escalation signal.
+* A fatal alert is issued. 
+  In this case, a full secure wipe is performed as a local action.
+
+Internal state secure wipe is automatically triggered on completion of the OTBN operation and after a recoverable error alert.
+
+Software can trigger a full state wipe by writing 3'b111 to {{< regref "SEC_WIPE">}}  (i.e. set all individual state wipe bits).
+
+#### Data Memory (DMEM) Secure Wipe {#design-details-secure-wipe-dmem}
+
+The wiping is performed by securely replacing the memory scrambling key, making all data stored in the memory unusable. 
+The key replacement is a two-step process:
+
+* Overwrite the 128b key of the memory scrambling primitive with randomness from URND.
+  This action takes a single cycle.
+* Request new scrambling parameters from OTP. 
+  The request takes multiple cycles to complete.
+
+Host software can initiate a data memory secure wipe by writing 1 to the {{< regref "SEC_WIPE.dmem">}} register field.
```

Comment:
Add a reference here to a section describing how completion of the operation is signaled.